hedera-node/configuration/testnet/bootstrap.properties

```diff
@@ -1,5 +1,10 @@
-ledger.id=0x01
```

Comment:
Does testnet use the same ledger ID as mainnet?

Reply:
No it doesn't. So I should probably put these all back.

Reply:
Reverted them back. Thanks Richard